bindings/fcl-js.go

```diff
@@ -0,0 +1,133 @@
+package bindings
+
+import (
+	"bytes"
+	"os"
+	"path/filepath"
+	"runtime"
+	"sort"
+	"text/template"
+
+	"github.com/onflow/flixkit-go"
+	"github.com/stoewer/go-strcase"
+)
+
+type simpleParameter struct {
+	Name        string
+	JsType      string
+	Description string
+	FclType     string
+	CadType     string
+}
+
+type templateData struct {
+	Version         string
+	Parameters      []simpleParameter
+	Title           string
+	Description     string
+	Location        string
+	IsScript        bool
+	IsLocalTemplate bool
+}
+
+type FclJSGenerator struct {
```

Comment:
should this be lowercased?

Reply:
It's publicly assessable. I thought it needed to be uppercase so calls can get to ti.

Reply:
It has to be uppercased for other packages to access it yes.